src/cijobs/Program.cs

```diff
@@ -69,15 +69,13 @@ public async Task<int> RunAsync(string name)
             return await CopyAsync(cic);
         }
 
-        private T Get<T>(Option<T> option) => _command.Result.GetValue(option);
-
-        private static Task<int> Main(string[] args) =>
-            new CommandLineBuilder(new CIJobsRootCommand(args))
-                .UseVersionOption("--version", "-v")
-                .UseHelp()
-                .UseParseErrorReporting()
-                .Build()
-                .InvokeAsync(args);
+        private T Get<T>(CliOption<T> option) => _command.Result.GetValue(option);
+
+        private static int Main(string[] args) =>
+            new CliConfiguration(new CIJobsRootCommand(args).UseVersion())
+            {
+                EnableParseErrorReporting = true
+            }.Invoke(args);
```

Comment:
We don't have a good story for it yet, but in general we want to avoid or even forbid the users to mix sync and async. Since all of the sub commands define async actions here, we should call `InvokeAsync` here as well

```suggestion
        private static Task<int> Main(string[] args) =>
            new CliConfiguration(new CIJobsRootCommand(args).UseVersion())
            {
                EnableParseErrorReporting = true
            }.InvokeAsync(args);
```

Reply:
Nice tip, I'll try to remember it. Thank you! :)